### PR TITLE
add refwidget

### PR DESCRIPTION
**Pull request by tschanzt**

### CODE REVIEW

File: sources.cfg [hunk 1]
```diff
@@ -10,3 +10,7 @@ auto-checkout =
     ftw.theming
 # https://github.com/4teamwork/ftw.colorbox/pull/8
     ftw.colorbox
+    ftw.referencewidget
+
+[branches]
+ftw.referencewidget = tt-impl
```
Comment (Tschanzt) [inline sources.cfg]: 🚧

File: sources.cfg [hunk 2]
```diff
@@ -10,3 +10,7 @@ auto-checkout =
     ftw.theming
 # https://github.com/4teamwork/ftw.colorbox/pull/8
     ftw.colorbox
+    ftw.referencewidget
```
Comment (Tschanzt) [inline sources.cfg]: 🚧

**Comment by maethu** [comment]
@Tschanzt Please use the first release

**Comment by Tschanzt** [comment]
@maethu updated.

**Comment by mbaechtold** [comment]
@Tschanzt, maybe you could squash these 3 commits? Or at least the first and third commit?

**Comment by Tschanzt** [comment]
@mbaechtold @maethu done

**Comment by mbaechtold** [comment]
@Tschanzt, could you please add an entry to the changelog?

**Comment by Tschanzt** [comment]
@mbaechtold updated.

**Comment by mbaechtold** [comment]
@Tschanzt, could please rebase onto `master`?

**Comment by Tschanzt** [comment]
@mbaechtold ⬆️

**Comment by mbaechtold** [comment]
Cool 👍 Looks good to me.

**Comment by mbaechtold** [comment]
@maethu, are you ready for the new reference widget?

**Comment by mbaechtold** [comment]
Cool, thank you very much, @tschanzt!

**Pull request merged**